tests/functional/TestSmokeTest/tests/heketi_test.go

```diff
@@ -264,7 +264,7 @@ func TestHeketiCreateVolumeWithGid(t *testing.T) {
 	volInfo, err := heketi.VolumeCreate(volReq)
 	tests.Assert(t, err == nil)
 
-	// SSH into system and execute gluster command to create a snapshot
+	// SSH into system and try creating files from user belonging to gid
```

Comment:
why the comment change here?
it breaks rebase to pr #614 and I think is too early in this commit?